### PR TITLE
Mention the need of shutdown/startup in update docs

### PR DESCRIPTION
**Pull request by Dav-Git**
### Type

- [ ] Bugfix
- [X] Enhancement
- [ ] New feature

### Description of the changes
Previously the update docs never mentioned that you have to restart yur bot for the update to take effect.

### CODE REVIEW

**Comment by Drapersniper** [comment]
I think, it would be better just telling them to Always complete the update with the bot off, this would avoid potential permissions error, and any other potential issue wit ha hot update.

**Comment by Dav-Git** [comment]
I'm not quite sure on the wording for 3.1.X and 3.0 and older...

File: docs/update_red.rst [hunk 1]
```diff
@@ -30,15 +30,17 @@ Red 3.2.0 or newer
 Windows
 -------
 
-If you have Red 3.2.0 or newer, all you have to do to upgrade Red is running 2 commands.
+If you have Red 3.2.0 or newer, upgrading can be achieved by following these 4 easy steps:
```
Comment (Jackenmen) [inline docs/update_red.rst]:
```suggestion
If you have Red 3.2.0 or newer, you can upgrade by following these 4 easy steps:
```

File: docs/update_red.rst [hunk 2]
```diff
@@ -48,12 +50,16 @@ If you have Red 3.2.0 or newer, all you have to do to upgrade Red is running 2 c
 
     If you're using PostgreSQL data backend, replace ``Red-DiscordBot`` in the second command with ``Red-DiscordBot[postgres]``
 
+4. Start your bot.
+
 Linux & Mac
 -----------
 
-If you have Red 3.2.0 or newer, all you have to do to upgrade Red is running 2 commands.
+If you have Red 3.2.0 or newer, upgrading can be achieved by following these 4 easy steps:
```
Comment (Jackenmen) [inline docs/update_red.rst]:
```suggestion
If you have Red 3.2.0 or newer, you can upgrade by following these 4 easy steps:
```

**Review by Jackenmen — CHANGES_REQUESTED** [review]
I'll commit the suggestions in a moment, looks good otherwise

**Review by Kowlin — APPROVED** [review]
LGTM

**Review by Jackenmen — APPROVED** [review]

**Pull request merged**